iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/FittingAffOrgSectionWordsMatchVoter.java

```diff
@@ -75,25 +79,40 @@ public FittingAffOrgSectionWordsMatchVoter(List<Character> charsToFilter, int wo
     //------------------------ LOGIC --------------------------
     
     /**
-     * Returns true if minFittingOrgWordsPercentage of {@link AffMatchOrganization#getName()} words
+     * Returns true if minFittingOrgWordsPercentage the words of at least one of the organization names
```

Comment:
Is `minFittingOrgWordsPercentage` reference to any variable? I couldn't find any of this name.

This is also referred in two other voters: `FittingAffOrgWordsMatchVoter` and `FittingOrgWordsMatchVoter`.